packages/config/config/devices/0x014f/ws15z-1.json

```diff
@@ -20,21 +20,33 @@
 	"paramInformation": {
 		"3": {
 			"label": "Night Light",
-			"description": "By default, the LED will turn OFF when the lead attached is turned ON.",
+			"description": "By default, the LED will turn OFF when the load attached is turned ON.",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
-			"defaultValue": 1,
+			"maxValue": 4,
+			"defaultValue": 0,
 			"unsigned": true,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "On when load is on",
+					"label": "OFF when load is on",
```

Comment:
Don't capitlize the entirety of ON and OFF. Just "On" and "Off". Please carry throughout all files.